src/samples/audio/nxp_detect_test.c

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2021 NXP
+//
+// Author: Cristina Feies <cristina.ilie@nxp.com>
+// Author: Viorel Suman <viorel.suman@nxp.com>
+
+#include <sof/samples/audio/nxp_detect_test.h>
+
+static __aligned(64) uint8_t preprocessed_data[CONFIG_PREPROCESSED_SIZE];
+static __aligned(8) uint8_t confidences[4];
+
+//#define PRINT
+
+int postprocess(uint8_t confidences[4])
+{
+	int i;
+	uint8_t max_confidence = confidences[0];
+	int max_idx = 0;
+
+	for (i = 1; i < 4; i++) {
+		if (max_confidence < confidences[i]) {
+			max_confidence = confidences[i];
+			max_idx = i;
+		}
+	}
+	if (max_confidence < 128)
```

Comment:
Another magic number, 128.  Why did you choose 128? It cannot be changed?
I believe you should use a define and explain why you use 128.